e2e/wdio/platform/pages/object-attribute.po.ts

```diff
@@ -0,0 +1,19 @@
+import { BaseComponentPo } from './base-component.po';
+import { waitForElDisplayed, waitForPresent } from '../../driver/wdio';
+
+export class ObjectAttributePo extends BaseComponentPo {
+    url = '/object-attribute';
+    root = '#page-content';
+
+    textExample = 'fdp-object-attribute-example ';
```

Comment:
you can have three selectors for all your cases and no concat in the spec.

Reply:
done